server.js

```diff
@@ -23,8 +23,7 @@ if (fs.existsSync('./.env')) { // Check if we're running on a local dev machine
 
 }
 
-const getHtml = require('./index.html.js'); // Ugly HTML template TODO: Do this better
-const html = getHtml();
+const html = fs.readFileSync('./index.html.js', 'utf-8');
```

Comment:
Should this still be here? Wasn't `index.html.js` deleted?

Reply:
This shows that I must have tested this with `yarn dev` instead of `yarn start`…